index.html

```diff
@@ -0,0 +1,71 @@
+          <li class="li-left"><a href="https://web.archive.org/web/20140301004610/http://www.apple.com/"><img src="https://www.mercadonegro.pe/wp-content/uploads/2017/04/f527ea6db08ad6b5f87c68927ca7835f.png" alt="apple" class="apple-logo"></a></li>
+          <li><a class="aClass" href="https://web.archive.org/web/20140301004610/http://store.apple.com/">Store</a></li>
+          <li><a class="aClass" href="https://web.archive.org/web/20140301004610/http://www.apple.com/mac/">Mac</a></li>
+          <li><a class="aClass" href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipod/">iPod</a></li>
+          <li><a class="aClass" href="https://web.archive.org/web/20140301004610/http://www.apple.com/iphone/">iPhone</a></li>
+          <li><a class="aClass" href="https://web.archive.org/web/20140301004610/http://www.apple.com/ipad/">iPad</a></li>
+          <li><a class="aClass" href="https://web.archive.org/web/20140301004610/http://www.apple.com/itunes/">iTunes</a></li>
+          <li><a class="aClass" href="https://web.archive.org/web/20140301004610/http://www.apple.com/support/">Support</a></li>
+          <li class="last-li li-right"><input type="text" name="search"></li>
+        </ul>
+      </nav>
+      <div class="p first-p">
+        <a class="extra-link" href="https://web.org/web/20140228175622/http://www.apple.com/your-verse/">What will your verse be?</a>
+      </div>
+      <div class="p second-p">
+        <a class="extra-link" href="https://web.org/web/20140228175622/http://www.apple.com/your-verse/">Everyone has something to share. See how people are using<br>
+        iPad in their unique ways to add to the world's story.</a>
+      </div>
+      <div class="p third-p">
+        <a class="extra-link extra-link-hover" href="https://web.archive.org/web/20140301004610/http://www.apple.com/your-verse/#video-your-verse">
+          Watch the “Your Verse” film and explore the stories <span class="glyphicon glyphicon-play-circle" aria-hidden="true"></span>
+        </a>
+
+      </div>
+    </header>
+    <main class="container">
+
+      <a href="https://web.org/web/20140228175622/http://www.apple.com/your-verse/">
+        <img  id="main-img" src="https://i.pinimg.com/originals/96/c9/b9/96c9b9a6ef54cd17bc906af66d98c51b.jpg" alt="img">
+      </a>
+      <img class="all-img-bottom" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s.jpg" alt="img">
+      <img class="img-center all-img-bottom" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c.jpg" alt="img">
+      <img class="img-center all-img-bottom" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years.jpg" alt="img">
+      <img class="img-center all-img-bottom" src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility.jpg" alt="img">
+
```

Comment:
* The images at the bottom should be within the footer. And they should be set as the background of a` <div> `tag, and at the same tame enclosed in `<a>` tags.